samples/Catalog/Catalog.android.js

```diff
@@ -119,6 +119,14 @@ var examples = [
       component.props.navigation.navigate("PdfViewInstantJsonScreen");
     }
   },
+  {
+    name: "Programmatic Form Filling",
+    description:
+      "Shows how to programatically read and write PDF forms.",
```

Comment:
> Shows how to programatically read and **write** PDF forms

The "write" part implies that forms are being created to me. Can we change this to mention this is about filling forms and reading filled values.

Also "programatically" should be "programmatically" (double 'm').